src/status_im/contexts/wallet/wallet_connect/session_proposal/view.cljs

```diff
@@ -60,13 +65,11 @@
   (let [accounts         (rf/sub [:wallet/accounts-without-watched-accounts])
```

Comment:
Please use the `wallet/operable-accounts-without-watched-accounts` sub to display only the operable accounts.